<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WifiPad Server</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="leServerName">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>381</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Server name:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="cbConfiguration">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>211</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bNewConfiguration">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>New...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bDeleteConfiguration">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bSaveConfiguration">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>51</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Configuration:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="cbGamepads">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>251</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bInstallGamepad">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>70</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Install...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bDeleteGamepad">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Gamepads:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="bShowTapZones">
    <property name="geometry">
     <rect>
      <x>362</x>
      <y>100</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show tap zones</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bShowKeys">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show keys</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="GamePadView" name="gvGamepadView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>480</width>
      <height>320</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>480</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>421</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Test button presses in this box below</string>
    </property>
   </widget>
   <widget class="QSlider" name="hsMouseSpeed">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>103</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Mouse speed:</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GamePadView</class>
   <extends>QGraphicsView</extends>
   <header>GamePadView.h</header>
   <slots>
    <slot>Refresh()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leServerName</tabstop>
  <tabstop>cbConfiguration</tabstop>
  <tabstop>bNewConfiguration</tabstop>
  <tabstop>bDeleteConfiguration</tabstop>
  <tabstop>bSaveConfiguration</tabstop>
  <tabstop>cbGamepads</tabstop>
  <tabstop>bInstallGamepad</tabstop>
  <tabstop>bDeleteGamepad</tabstop>
  <tabstop>hsMouseSpeed</tabstop>
  <tabstop>bShowKeys</tabstop>
  <tabstop>bShowTapZones</tabstop>
  <tabstop>gvGamepadView</tabstop>
  <tabstop>lineEdit_2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>leServerName</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>ServerNameChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>17</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbGamepads</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>ChangeGamePad(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bShowKeys</sender>
   <signal>stateChanged(int)</signal>
   <receiver>gvGamepadView</receiver>
   <slot>Refresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bShowTapZones</sender>
   <signal>stateChanged(int)</signal>
   <receiver>gvGamepadView</receiver>
   <slot>Refresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbConfiguration</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>ChangeConfiguration(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>19</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bDeleteConfiguration</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>DeleteConfiguration()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>497</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bSaveConfiguration</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>SaveConfiguration()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>472</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bNewConfiguration</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>NewConfiguration()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bInstallGamepad</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>InstallGamepad()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bDeleteGamepad</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>DeleteGamepad()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hsMouseSpeed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>SetMouseSpeed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ServerNameChanged(QString)</slot>
  <slot>ChangeGamePad(int)</slot>
  <slot>ChangeConfiguration(int)</slot>
  <slot>DeleteConfiguration()</slot>
  <slot>SaveConfiguration()</slot>
  <slot>NewConfiguration()</slot>
  <slot>InstallGamepad()</slot>
  <slot>DeleteGamepad()</slot>
  <slot>SetMouseSpeed(int)</slot>
 </slots>
</ui>
